server/src/main/java/org/apache/kafka/server/metrics/BrokerServerMetrics.java

```diff
@@ -89,12 +93,18 @@ public BrokerServerMetrics(Metrics metrics) {
                 METRIC_GROUP_NAME,
                 "The number of errors encountered by the BrokerMetadataPublisher while applying a new MetadataImage based on the latest MetadataDelta."
         );
+        ignoredStaticVotersName = metrics.metricName(
+            "ignored-static-voters",
+            METRIC_GROUP_NAME,
+            "1 if controller.quorum.voters is set but was not used by the broker, 0 otherwise."
+        );
 
         metrics.addMetric(lastAppliedRecordOffsetName, (config, now) -> lastAppliedImageProvenance.get().lastContainedOffset());
         metrics.addMetric(lastAppliedRecordTimestampName, (config, now) -> lastAppliedImageProvenance.get().lastContainedLogTimeMs());
         metrics.addMetric(lastAppliedRecordLagMsName, (config, now) -> now - lastAppliedImageProvenance.get().lastContainedLogTimeMs());
         metrics.addMetric(metadataLoadErrorCountName, (config, now) -> metadataLoadErrorCount.get());
         metrics.addMetric(metadataApplyErrorCountName, (config, now) -> metadataApplyErrorCount.get());
+        metrics.addMetric(ignoredStaticVotersName, (Gauge<Boolean>) (config, now) -> ignoredStaticVoters.get());
```

Comment:
According to the KIP is metrics is expected to have the values 1 and 0, not true and false.